SECURITY.MD

```diff
@@ -0,0 +1,19 @@
+# O3DE Security Policy
+
+## Reporting a Vulnerability
+
+If you have information about a security issue or vulnerability in O3DE, please send an e-mail to [security@o3de.org](mailto:security@o3de.org).
+
+Please provide as much information as possible, including:
+
+- All relevant fields from the O3DE standard [issue template](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md).
+
+- A detailed description of the vulnerability we can use to reproduce your findings.
+
+- A definition of who can exploit this vulnerability and what they would gain.
+
+- Information about any known exploits.
+
+A member of the [SIG-Security](https://github.com/o3de/sig-security/) Issue Response Team will review your e-mail and contact you to collaborate on resolving the issue.
+
+For more details, please refer to the [Security Documentation](https://www.o3de.org/docs/contributing/security) for O3DE.
```

Comment:
This link is currently dead, but I see there's an open PR at https://github.com/o3de/o3de.org/pull/1431

Recommend placing a link to this documentation near the top of this file, as it provides specific guidance and should probably be the first thing the audience looks at. This file appears to be trying to cover similar information, which makes me wonder how much needs to be spelled out in this file. Directions in this file may be useful when full documentation is unavailable for any reason, but I think we should avoid duplicating all but the simplest steps here. May want to clarify that the steps here are shorthand for the ones in the documentation.

Reply:
This is supposed to be an "at a glance" document and to have enough details so you don't have to go to the full docs.

Link is currently dead and PR won't be pushed until docs page is live.

Reply:
I can add a link to the documentation near the "reporting issue" section as well.